package.json

```diff
@@ -140,13 +136,9 @@
     "metalsmith-redirect": "^2.1.0",
     "metalsmith-sitemap": "^1.0.0",
     "metalsmith-watch": "^1.0.3",
-    "metalsmith-webpack": "department-of-veterans-affairs/metalsmith-webpack",
-    "metalsmith-webpack-dev-server": "department-of-veterans-affairs/metalsmith-webpack-dev-server",
     "mkdirp": "^0.5.1",
```

Comment:
Actually never realized til now that we maintained the Metalsmith plugin for Webpack dev server.

Reply:
Actually, I didn't either. @bshyong, do you think we should continue updating those repositories or move the code into vets-website like I did here?

Reply:
@jbalboni I believe we forked these because I made some changes to those libraries to work with vets-website (will have to diff to see what the changes were).

I think it’s fine to move them into our repo instead of using a fork.

Reply:
It looks like the only change was to update the webpack dependency version.